zetaclient/config/types.go

```diff
@@ -91,8 +91,10 @@ type Config struct {
 
 	// chain configs
 	EVMChainConfigs map[int64]EVMConfig `json:"EVMChainConfigs"`
-	BitcoinConfig   BTCConfig           `json:"BitcoinConfig"`
-	SolanaConfig    SolanaConfig        `json:"SolanaConfig"`
+	BTCChainConfigs map[int64]BTCConfig `json:"BTCChainConfigs"`
+	// Deprecated: the 'BitcoinConfig' will be removed once the 'BTCChainConfigs' is fully adopted
+	BitcoinConfig BTCConfig    `json:"BitcoinConfig"`
```

Comment:
Can we make sure that `.BitcoinConfig` is not used anywhere except `GetBTCConfig(chainID)`?

Reply:
Yep. That's right.